### PR TITLE
fix(webpack-manifest): Manifest plugin errors out if manifest.json exists

### PR DESCRIPTION
**Pull request by dac09**
## What does this do?

Since the webpack 5 upgrade, if your project has a manifest.json in `web/public/manifest.json` - building web fails with

```
ERROR in Conflict: Multiple assets emit different content to the same filename manifest.json
```

This fix renames the file that this plugin generates to `rw-build-manifest.json`, and also updates the prerender code to look for the new file in the img/asset processing plugin.

---

### WTH is manifest?!
manifest.json is typically used for PWAs. In our case, its actually serving a different purpose in prerender

### CODE REVIEW

File: packages/core/config/webpack.common.js [hunk 1]
```diff
@@ -226,7 +226,10 @@ module.exports = (webpackEnv) => {
           // @TODO: Add redirect to fatalErrorPage
           // lastResortScript: "window.location.href='/500.html';"
         }),
-      isEnvProduction && new WebpackManifestPlugin(),
+      isEnvProduction &&
+        new WebpackManifestPlugin({
+          fileName: 'rw-build-manifest.json',
```
Comment (thedavidprice) [inline packages/core/config/webpack.common.js:231]: how about just 'build-manifest.json`. the `rw` feels redundant, yes?

**Comment by thedavidprice** [comment]
@dac09 I had a small naming question. Otherwise, if this is working for you, please merge away!

Lastly, please do help label appropriately for Release Notes breaking and code mods. 👍

File: packages/core/config/webpack.common.js (hunk 1 above)
Comment (dac09) [reply packages/core/config/webpack.common.js]: You got it!

**Comment by dac09** [comment]
Hey @thedavidprice good point - but this isn't a breaking change :), but without it webpack 5 _could_ be a breaking change.

**Review by peterp — APPROVED** [review]

**Comment by thedavidprice** [comment]
> But without it webpack 5 could be a breaking change.

^^ oh, Webpack is a breaking change

**Pull request merged**